src/azure-cli-testsdk/azure/cli/testsdk/preparers.py

```diff
@@ -170,6 +170,18 @@ def remove_resource(self, name, **kwargs):
         if not self.dev_setting_sp_name:
             execute(self.cli_ctx, 'az ad sp delete --id {}'.format(self.result['appId']))
 
+# Function wise, enabling large payload recording has nothing to do with resource preparers 
+# We still base on it so that this decorator can chain with other preparers w/o too much hacks
+class LargeResponsePreparer(AbstractPreparer):
+    def __init__(self, size_kb=1024):
+        self.size_kb = size_kb
+        super(LargeResponsePreparer, self).__init__('nanana', 20)
```

Comment:
nanana?

Reply:
3 `n/a` to make the `AbstractPreparer` happy on the random name length. :)

Reply:
it is not really used, but a workaround to get in the decorator's chain